src/Files.App/Program.cs

```diff
@@ -62,11 +62,17 @@ private static void Main()
 			var OpenTabInExistingInstance = ApplicationData.Current.LocalSettings.Values.Get("OpenTabInExistingInstance", true);
 			var activatedArgs = AppInstance.GetCurrent().GetActivatedEventArgs();
 
-			if (activatedArgs.Data is ICommandLineActivatedEventArgs cmdLineArgs)
+			// WINUI3: When launching from commandline the argument is not ICommandLineActivatedEventArgs (#10370)
```

Comment:
I think this issue has been resolved.
Can you please confirm this code is still valid

Reply:
Yeah I've checked and this is still needed. Equivalent code is in MainWindow.cs

Reply:
Thanks for the checking.